content/tutorials/configuration.md

```diff
@@ -113,6 +114,42 @@ tls:
 Optionally, also configure a TLS proxy in this section. 
 See the separate [TLS Proxy page]({{< relref "concepts/tls-proxy" >}}) for more information.
 
+#### API Configuration
+
+By default, KES requires a valid TLS certificate for all API calls.
+You can change this behavior for specific API endpoints to allow requests to the endpoints without a certificate.
+
+{{< admonition type="caution" >}}
+The default behavior of the KES API endpoints should suffice in most every situation.
+Only customize endpoints for specific need
+{{< /admonition >}}
+
+When you disable authentication for at least one endpoint, KES no longer requires a certificate for a client to call _any_ API endpoint.
```

Comment:
Ohhhh this is why I was confused about when the cert is required. Making cert not required for one _does_ change behavior for others. Hence my other comment. Maybe this could be more prominent. (Admonition seems a little much however.)

Reply:
I agree that an admonition is not needed here. Going to leave as is for now unless an idea for highlighting it further comes to mind.